config/routes.rb

```diff
@@ -1,6 +1,9 @@
 FinderFrontend::Application.routes.draw do
   mount GovukPublishingComponents::Engine, at: "/component-guide"
 
+  get "/sign-in", to: proc { [200, {}, %w[OK]] }
+  get "/sign-out", to: proc { [200, {}, %w[OK]] }
```

Comment:
Could these be moved next to the `get "/email/subscriptions/new", to: proc { [200, {}, [""]] }, as: :email_alert_frontend_signup`, and with a comment above them saying something like `# Needed for the brexit checker tests`?